doc/deployment_guide_s390.md

```diff
@@ -0,0 +1,147 @@
+
+```bash
+FTP example.org (addr ipv4 
+anonymous 
+
+cd boot/s390x
+locsite fix 80
+ascii
+get parmfile sles.parmfile.a (repl
+get sles.exec sles.exec.a (repl
+locsite fix 80
+binary
+get linux sles.linux.a (repl 
+get initrd sles.initrd.a (repl 
+quit
+```
+
+**Note:** the command `locsite fix 80`, which sets the VM file format to fixed length 80, the file format necessary for punching the binary files to the virtual machine reader.
+
+You can use the FILELIST command to list the files and edit the **parmfile** if needed with XEDIT.
+
+For our machine the parmfile is:
+
+```txt
+cio_ignore=all,!condev
+rd.cio_accept=0.0.0160
+rd.zdev=qeth,0.0.0800:0.0.0801:0.0.0802,layer2=1,portno=0
+ip=192.168.0.111::192.168.0.1:24:zvmtest.example.org:enc800:none
+nameserver=192.168.0.1
+root=live:http://example.org/d-installer.iso
+```
+
+Although **cio_ignore** parameter is optional we used it in order to list only the relevant installation devices accepting the ones which we will use for the installation. 
+
+As we do not have an interactive dialog for enabling and configuring our network device we need to provide the settings through the kernel command line, in this case we will use rd.zdev option for activating our qeth device and the ip option for configuring network settings the enc800 linux network interface.
+
+We even could activate our DASD device with the **rd.zdev=dasd,0.0.0160** option which in that case the **rd.cio_accept=0.0.0160** could be omitted as it is superfluous but for this documentation we will do it later with D-Installer.
+
+Finally we will boot from a live image retrieved from an url using the **root=live:<url>** parameter.
+  
+The content of the sles.exec file is:
+
+```txt
+/* REXX LOAD EXEC FOR SUSE LINUX S/390 VM GUESTS       */
+/* LOADS SUSE LINUX S/390 FILES INTO READER            */
+SAY ''                                                   
+SAY 'LOADING SLES FILES INTO READER...'                  
+'CP CLOSE RDR'                                           
+'PURGE RDR ALL'                                          
+'SPOOL PUNCH * RDR'                                      
+'PUNCH SLES LINUX A (NOH'                                
+'PUNCH SLES PARMFILE A (NOH'                             
+'PUNCH SLES INITRD A (NOH'                               
+'IPL 00C'                                                
+```
+
+Once ready boot the installation running the **sles.exec** REXX file
+
+```bash
+sles
+```
+
+Once the installation system finish the booting process just connect to the machine with the web browser (e.g. https://s390vsl111.suse.de:9090) or by SSH with (root / linux) user.
+
+## Complete installation Z/VM installation workflow screenshots
+
+For taking the screenshots we have omitted the cio_ignore parameter from the parmfile and as dhcp config is supported the config is quite simple as we can see below:
```

Comment:
Maybe we can add some comes here somewhere to makes this sentence parseable. :-)  The "bla as bla as bla" structure is weird.